posthog/session_recordings/session_recording_api.py

```diff
@@ -682,10 +666,52 @@ def _stream_blob_to_client(
                 response["Cache-Control"] = streaming_response.headers.get("Cache-Control") or "max-age=3600"
 
                 response["Content-Type"] = "application/json"
-
                 response["Content-Disposition"] = "inline"
+
                 return response
 
+    def _send_realtime_snapshots_to_client(
+        self, recording: SessionRecording, request: request.Request, event_properties: dict
+    ) -> HttpResponse | Response:
+        version = request.GET.get("version", "og")
```

Comment:
`og` is the fallback? or dies it exist and is the default? :D

Reply:
yep, it's the fallback for when we receive no version in the URL :)

we'll be able to remove this entire branch once we're confident all or most clients will have refreshed since this released